pkg/api/Helper.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+const dateformat = "20060102"
+const limit = 50
+const DBdateFormat = "02.01.2006"
+
+func writeJson(w http.ResponseWriter, code int, data any) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(code)
+	jsonData, _ := json.Marshal(data)
+	w.Write(jsonData)
```

Comment:
Лучше не игнорировать ошибку, а просто вывести ее в лог
```suggestion
	_, err := w.Write(jsonData)
```